src/business-logic/services/test/device-changed-service.spec.ts

```diff
@@ -94,13 +94,13 @@ describe(DeviceChangedService.name, () => {
 
 function createTestee(params?: {
   statusMessageService?: StatusMessageService,
-  deviceRepository?: DeviceRepository,
-  deviceDataChangedListener?: DataChangedListener<Device>,
+  deviceRepository?: CoreDeviceRepository,
+  deviceDataChangedListener?: DataChangedListener<CoreDevice>,
   logger?: Logger
 }): DeviceChangedService {
-  let deviceRepository: DeviceRepository
+  let deviceRepository: CoreDeviceRepository
   if (!params?.deviceRepository) {
-    const deviceRepositoryMock: DeviceRepository = mock<DeviceRepository>()
+    const deviceRepositoryMock: CoreDeviceRepository = mock<CoreDeviceRepository>()
```

Comment:
Should still be `DeviceRepository`, we shouldn't have an interface called `CoreDeviceRepository`